packages/teams-js/src/public/call.ts

```diff
@@ -64,6 +64,7 @@ export namespace call {
   }
 
   export function isSupported(): boolean {
```

Comment:
>isSupported [](http://example.com/codeflow?start=18&length=11)

I know that you aren't making this any worse and that generally I don't see the need to fix up other code in the area that you're making a change, but I wonder if now would be the time to put documentation on these methods, since they are simple and you are in here already?

If you accept my previous "@throws" suggestion then that might be motivation to do it in this change.

Reply:
I can do that, it might slow down when this change gets checked in though. It will take a little while to add and check comments on however many functions that is. We can talk about it tomorrow if I haven't done it by then

Reply:
Done